### ISSUE TITLE
Add support for stat scrapers

### ISSUE THREAD

**Issue by npearce** [npearce/n8-BigStats#70]
BigStats is a stats exporter, designed around pushing data to stats databases/messages buses. However, some systems insist on old-school device polling.

Investigate allowing stats scrapers to `GET /mgmt/shared/bigstats` and retrieve the BigStats 'Stats Object' themselves.

**Comment by npearce** [comment]
`POST /mgmt/shared/bigstats_settings`

```json
{
  "config":  {
    "destination": "none"
  },
  "size": "[small|medium|large]",
  "interval": 10,
  "enabled": true
}
```

**Comment by npearce** [comment]
Implemented in v0.5
